.babelrc

```diff
@@ -25,13 +25,19 @@
           ]
         },
         "useBuiltIns": "entry",
-        "corejs": "2.0.0",
+        "corejs": "3.8.1",
         "modules": false,
         "debug": false
       }
     ],
     "@babel/react"
   ],
+  "ignore": [
```

Comment:
This is the change compared to the original PR https://github.com/department-of-veterans-affairs/vets-website/pull/15282

Reply:
Looks another difference is that this the PR has no changes in `preESModulesPolyfills.js`. Am I correctly interpreting that having Babel ignore the file resulted in the same list of polyfills when you followed that process to regenerate it?

Reply:
This PR has a bunch of changes to preESModulesPolyfills.js, but GitHub is hiding those by default due to the large diff so it's easy to miss. The changes are small naming changes due to the core-js upgrade, and also the addition of new polyfills which were automatically added when regenerating. All the old polyfills are still present as well though

Reply:
Okay, I did miss that the first time around. I see it now. 👀